### PR TITLE
Readme / Links update

### PR DESCRIPTION
**Pull request by ChrisCoastal**
@Zenoo Here are the changes we discussed:

- updated the link in `Header.tsx` to point to this repo
- updated author / contact info throughout to reflect your maintenance of v5
- added a legacy section for MUI v4 users / acknowledgement to Brian Reynolds in the `README.md`
- removed commented out Gatsby instructions in `README.md`

Let me know of any changes. Thanks!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,8 +2,9 @@
   "name": "material-ui-theme-creator",
   "private": true,
   "description": "A comprehensive tool to customize themes for Material-UI",
-  "version": "4.11.0",
-  "author": "Brian Reynolds <brireyn@gmail.com>",
+  "version": "5.0.0",
+  "original-by": "Brian Reynolds <brireyn@gmail.com>",
+  "v5-author": "Zenoo <https://github.com/Zenoo>",
```
Comment (Zenoo) [inline package.json]: Those fields don't exist inside `package.json`.  
Instead, you could use the [`author` and `contributors` fields](https://classic.yarnpkg.com/lang/en/docs/package-json/#toc-maintainers).

**Review by Zenoo — CHANGES_REQUESTED** [review]

**Comment by ChrisCoastal** [comment]
Thanks for the link/resource! I'd not yet looked at a resource for the `package.json` properties (clearly). 😅 ✅

I amended with additional fields `keywords` and `homepage` also in `package.json`

**Review by Zenoo — APPROVED** [review]
Everything seems good, thanks for your contributions!

**Comment by ChrisCoastal** [comment]
My pleasure. Thanks for all the work updating the tool for v5!

**Pull request merged**